### PR TITLE
[OUI docs] Keypad: change beta to experimental

### PR DESCRIPTION
**Pull request by ashwin-pc**
### Description
Changes Keypad beta references to experimental
 
### Issues Resolved
[(OUI Docs) Key pad menu#200](https://github.com/opensearch-project/oui/issues/200)
 
### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] All tests pass
  - [ ] `yarn lint`
  - [ ] `yarn test-unit`
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by KrooshalUX** [comment]
@ashwin-pc  can you upload a screenshot of the visual result for my UX approval?

**Review by seanneumann — APPROVED** [review]
LGTM

**Review by BSFishy — APPROVED** [review]

**Comment by BSFishy** [comment]
Waiting for UX approval

**Pull request merged**
